### PR TITLE
pull handle only

### PR DESCRIPTION
**Pull request by hljoren**
Pull the handle only, rather than the whole user object

### CODE REVIEW

File: twitter_user_handles.py [hunk 1]
```diff
@@ -38,11 +45,13 @@
 
 # Search each name, and write the first found verified user to the output json file
 
-    with open(out_dir + "/" + out_file + '.json', 'a') as outFile:
+    # with open(out_dir + "/" + out_file + '.json', 'a') as outFile: # entire user object
```
Comment (emmaling27) [inline twitter_user_handles.py]: remove this commented out line

File: twitter_user_handles.py [hunk 2]
```diff
@@ -38,11 +45,13 @@
...
+    with open(out_dir + '/candidate_handles.txt', 'a') as outFile: # handle only
         for name in data.split(','):
             if name != '':
                 query = name + '& filter:verified'
                 users = api.search_users(q=name)
                 if len(users) > 0:
                     user = users[0]
-                    json.dump(user._json, outFile)
+                    outFile.write('@' + user.screen_name + '\n') # handle only
+                    # json.dump(user._json, outFile) # entire user object
```
Comment (emmaling27) [inline twitter_user_handles.py]: also remove this commented out line

**Review by emmaling27 — CHANGES_REQUESTED** [review]
removing commented out code is probably best so people don't get confused

**Comment by hljoren** [comment]
All set!

**Review by jimwaldo — APPROVED** [review]

**Review by emmaling27 — APPROVED** [review]

**Pull request merged**